### ISSUE TITLE
Fix remaining advanced search bugs

### ISSUE THREAD

**Issue by code-arman** [shstrophies/SHSTrophies#29]
I did some thorough testing with advanced search. Here is what I found:

In every section EXCEPT for the "Years" section, if you put in something with a comma and then something else, it says that the input is invalid. For example: "Joe, Jeff", says input is invalid. But, we want this to return all trophies with joe and all of them with jeff. I've attached images below. The Years one does work correctly. If you do not want to support these searches in advanced search, let me know so that in the input for the "Name" text box, we dont say "or enter multiple names comma seperated"

![IMG_1525](https://user-images.githubusercontent.com/46976119/83102995-a72dde00-a06a-11ea-8653-5451ff3f4ccd.jpg)
![IMG_1523](https://user-images.githubusercontent.com/46976119/83103008-b14fdc80-a06a-11ea-929a-213e9b5ef96b.JPG)

 
![IMG_1526](https://user-images.githubusercontent.com/46976119/83102973-9aa98580-a06a-11ea-9e57-0b4f9bea3c96.jpg)

**Comment by code-arman** [comment]
I did some thorough testing with advanced search. Here is what I found:

In every section EXCEPT for the "Years" section, if you put in something with a comma and then something else, it says that the input is invalid. For example: "Joe, Jeff", says input is invalid. But, we want this to return all trophies with joe and all of them with jeff. I've attached images below. The Years one does work correctly
![IMG_1525](https://user-images.githubusercontent.com/46976119/83102995-a72dde00-a06a-11ea-8653-5451ff3f4ccd.jpg)
![IMG_1523](https://user-images.githubusercontent.com/46976119/83103008-b14fdc80-a06a-11ea-929a-213e9b5ef96b.JPG)

 
![IMG_1526](https://user-images.githubusercontent.com/46976119/83102973-9aa98580-a06a-11ea-9e57-0b4f9bea3c96.jpg)
